.github/workflows/build.yml

```diff
@@ -77,7 +77,6 @@ jobs:
         uses: ./.github/workflows/tmate_debug
         with:
           use-tmate: ${{ secrets.USE_TMATE }}
-          limit-access-to-actor: true
 
```

Comment:
I remember after adding this it fixed our problem and we confirmed
https://github.com/rook/rook/pull/12311#issuecomment-1570536295

Is smtg changed?

Reply:
that is not right fix, since by default `limit-access-to-actor: true` it is true only and in above pr the fix is not making effect as any tmate setting need to be in `.github/tmate_debyg/action.yaml` where we configure and the tmate.

In generally, if we have a composite yaml, all the configuration related to composite yaml file only,

Also, if you need now `web` access link is showing earlier it was not showing